map-prototype/nodes.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "FeatureCollection",
+  "features": [
```

Comment:
I guess we can [write our node-list schema](https://github.com/tomeshnet/node-list/issues/1) based on this?